NodeTypes/Content/ContactForm/ContactForm.yaml

```diff
@@ -24,6 +24,16 @@
           label: i18n
           icon: comment
           tab: default
+  references:
+    redirect:
+      constraints:
+        maxItems: 1
+      ui:
+        label: i18n
+        inspector:
+          group: redirect
+          editorOptions:
+            nodeTyps: [ 'Neos.Neos:Document' ]
```

Comment:
this here should also be moved to the `constraints` as the cr actually understands that but i forgot if the neos ui can handle it already :)